lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -46,14 +46,22 @@ def call(env)
           # configure the Rack middleware once
           configure()
 
-          # start a new request span and attach it to the current Rack environment;
-          # we must ensure that the span `resource` is set later
-          request_span = @tracer.trace(
-            'rack.request',
+          trace_options = {
             service: @service,
             resource: nil,
             span_type: Datadog::Ext::HTTP::TYPE
-          )
+          }
+
+          # Merge distributed trace ids if present
+          unless env['HTTP_X_DDTRACE_PARENT_TRACE_ID'].nil? || env['HTTP_X_DDTRACE_PARENT_SPAN_ID'].nil?
```

Comment:
Have these headers been documented as the "canonical" headers to use for distributed traces? I've seen various values used throughout clients for different languages.

Would be nice if there was a strict contract for this between all languages.